spec/vast_trackers.spec.js

```diff
@@ -223,5 +223,36 @@ describe('VASTTracker', function() {
         expect(spyEmitter).toHaveBeenCalledTimes(1);
       });
     });
+
+    describe('setProgress', () => {
+      let spyTrack;
+      beforeEach(() => {
+        spyTrack = jest.spyOn(vastTracker, 'track');
+        vastTracker.assetDuration = 10;
+        vastTracker.setProgress(5);
+      });
+      it('should be defined', () => {
+        expect(vastTracker.setProgress).toBeDefined();
+      });
+      it('call track with progress-5', () => {
+        expect(spyTrack).toHaveBeenCalledWith('progress-5', expect.anything());
```

Comment:
Do we actually care about the parameter being passed to the function ? Otherwise we can use toHaveBeenCalled() probably. Same with other tests below

Reply:
For the record, as setProgress doesn't have any unit tests, we do care about the parameter. We must check if tracker are called with the expected values for percentages format/time format.